### PR TITLE
get-started-tutorial-2 | table value format updates

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/use-azure-cli-successfully-quoting.md [hunk 1]
```diff
@@ -128,7 +131,71 @@ PowerShell don't.
   az ad app create --display-name myName --native-app --required-resource-accesses @manifest.json
   ```
 
-If you are working in the PowerShell scripting language, see [Considerations for running the Azure CLI in a PowerShell scripting language - JSON strings](./use-azure-cli-successfully-powershell.md#pass-parameters-containing-json).
+Here are the accepted JSON format patterns for Bash, PowerShell and Cmd:
+
+### [Bash](#tab/bash2)
+
+Use Bash's `clear` command to remove console output between tests.
```
Comment (jiasli) [inline docs-ref-conceptual/use-azure-cli-successfully-quoting.md:138]: I don't think it is necessary to mention `clear` in CLI's documentation.

File: docs-ref-conceptual/use-azure-cli-successfully-quoting.md [hunk 2]
```diff
@@ -113,13 +118,11 @@ PowerShell don't.
 
 ## JSON strings
 
-* If your command is only going to run on Bash (or Zsh), use single quotes to preserve the content
-  inside the JSON string. Single quotes are necessary when supplying inline JSON values. For
-  example, this JSON is correct in Bash: `'{"key": "value"}'`.
+* Use single quotes to preserve the content inside a JSON string. Single quotes are necessary when supplying inline JSON values. For example, this JSON is correct in both Bash and PowerShell: `'{"key": "value"}'`.
+
+* If your command runs at a Windows Command Prompt, you must use double quotes. The equivalent of the above JSON string in Cmd.exe is `"{"key":"value"}" `.
```
Comment (jiasli) [inline docs-ref-conceptual/use-azure-cli-successfully-quoting.md:123]: The correct command should be `"{\"key\":\"value\"}" `.

File: docs-ref-conceptual/use-azure-cli-successfully-quoting.md [hunk 3]
```diff
@@ -128,7 +131,71 @@ PowerShell don't.
...
+# Correct in Windows Command 
+az "{"key":"value"}" --debug
+>> Command arguments: ['{key:value}', '--debug']
+```
+
+These next three examples are all **incorrect** in Windows Command. 
+
+|Incorrect format|Problem |Console output|
+|-|-|-|
+|az "{"key":"value"}" --debug | Missing escape characters | Command arguments: ['{key:value}', '--debug']
```
Comment (jiasli) [inline docs-ref-conceptual/use-azure-cli-successfully-quoting.md:194]: This incorrect `"{"key":"value"}"` is the same as the above `# Correct in Windows Command` command. This makes this doc self-contradictory.

**Comment by jiasli** [comment]
This document contains errors and are not aligned with https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md

**Comment by dbradish-microsoft** [comment]
@jiasli, thank you for your feedback. This is azDO [#350797](https://dev.azure.com/msft-skilling/Content/_workitems/edit/350797/).

**Pull request merged**